android-components/components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestions.kt

```diff
@@ -57,6 +66,38 @@ internal fun Suggestions(
             }
         }
     }
+
+    val currentSuggestions by rememberUpdatedState(suggestions)
+
+    LaunchedEffect(Unit) {
```

Comment:
GitHub munches comments on individual commits after a force-push (which makes sense, but I didn't think about that—should've opened the draft PR a lot sooner! 😅), but here are breadcrumbs to previous conversations:

* @MatthewTighe's comment about testing, and my explanation of why I went with this approach: https://github.com/linabutler/firefox-android/commit/6055b58d7cc5c0b87112923133a073679767ef3f#r129642181
* Our conversations from an even earlier iteration, exploring different alternatives and figuring out how they work with Compose: https://github.com/mozilla-mobile/firefox-android/commit/e9d0ea629a5083f3b0bb1c3e3ca38921feeb46d2#r129168885, https://github.com/linabutler/firefox-android/commit/69a30b3641e3b2e340ee936fb45ada76aaa313ac#r129142449